core/domain/suggestion_services.py

```diff
@@ -3936,22 +3937,30 @@ def _generate_translation_contributor_certificate_data(
     words_count = 0
     for model in suggestions:
         suggestion = get_suggestion_from_model(model)
+        suggestion_change = suggestion.change
+        if (
+                suggestion_change.cmd == 'add_written_translation' and (
```

Comment:
This looks quite unwieldy, I suggest you make a new named variable to store the result of  is_data_format_list() and reference it here.

Reply:
Done